dts/bindings/sensor/adi,adxl372-common.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2022 Analog Devices Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    odr:
+      type: int
+      required: false
+      default: 0
+      description: |
+            Accelerometer sampling frequency (ODR)
+      enum:
+        - 0 #400Hz
+        - 1 #800Hz
+        - 2 #1600Hz
+        - 3 #3200Hz
+        - 4 #6400Hz
```

Comment:
Please document the encodings in the description so they get included in the rendered documentation. https://docs.zephyrproject.org/latest/build/dts/api/bindings.html